sample.env

```diff
@@ -32,6 +32,7 @@ REACT_APP_KEEPKEY_VERSIONS_URL=https://raw.githack.com/keepkey/keepkey-updater/m
 REACT_APP_WALLET_MIGRATION_URL=https://wallets.shapeshift.com/api/migrate
 REACT_APP_MIDGARD_URL=https://thor-midgard.shapeshift.com/v2
 REACT_APP_COWSWAP_HTTP_URL=https://api.cow.fi/mainnet/api
+REACT_APP_YAT_NODE_URL=https://octopus-app-mkjlj.ondigitalocean.app
```

Comment:
please rebase/merge develop and put this in `.env.base` - this file doesn't exist any more